server/service/src/processors/transfer/requisition/mod.rs

```diff
@@ -159,6 +159,24 @@ pub(crate) fn process_requisition_transfers(
 #[error("Database error in processor ({0}) {1:?}")]
 pub(crate) struct ProcessorError(String, RepositoryError);
 
+#[derive(Debug)]
+enum RequisitionTransferOutput {
```

Comment:
The story here (and for invoice processor equivalent) is that when I was testing my tests I realised that I had a false pass because a different check processor was preventing the transfer, rather than the one I had added. With this we can test precisely why the processor skipped. No substantial use for it in actual runtime code though.

Also tempted to do something funky like a hierarchy of enums where each processor has it's own enum of failure modes, but there is a fair bit of overlap between them and 1-big-enum is sorta easier to throw around anyway.

Reply:
siiick love the clearer skipping types

Reply:
> The story here (and for invoice processor equivalent) is that when I was testing my tests I realised that I had a false pass because a different check processor was preventing the transfer, rather than the one I had added. With this we can test precisely why the processor skipped. No substantial use for it in actual runtime code though.

I think in this case checking that record is created first, and then adjusting picked date/pref/initialisation date and, then doing another test would be the way to go